documentation/IDTA-01001/modules/Constraints/constraints.adoc

```diff
@@ -8,12 +8,12 @@ SPDX-License-Identifier: CC-BY-4.0
 ////
 
 // Constraints
-:aasd002: pass:q[[underline]#Constraint AASd-002:# _idShort_ of __Referable__s shall only feature letters, digits, underscore ("_"); starting mandatory with a letter, i.e. [a-zA-Z][a-zA-Z0-9_]*.]
+:aasd002: pass:q[[underline]#Constraint AASd-002:# _idShort_ of __Referable__s shall only feature letters, digits, hyphen ("-") and  underscore ("_"); starting mandatory with a letter, and not ending with a hypen or underscore, i.e. ^[a-zA-Z][a-zA-Z0-9_-]*[a-zA-Z0-9_]+$.]
```

Comment:
> ^[a-zA-Z][a-zA-Z0-9_-]*[a-zA-Z0-9_]+$.]

Good eye!

Reply:
Small nitpick, currently this allows the use of an underscore as a last character, when in the sentence before it is stated, that it shouldn't end with an underscore.

Reply:
Thank you for the finding. Unfortunately, we need to be backward compatible and it seems we allowed "_" as last char before. So we need to fix the text.